tests/Silverback.Integration.Kafka.Tests/Messaging/Messages/KafkaKeyHelperTests.cs

```diff
@@ -11,6 +11,26 @@ namespace Silverback.Tests.Integration.Kafka.Messaging.Messages
 {
     public class KafkaKeyHelperTests
     {
+        [Fact]
+        public void GetMessageKey_NullMessage_NullIsReturned()
```

Comment:
What convention do you use for naming?

Reply:
`MethodName_StateUnderTest_ExpectedBehavior`, your method name looks good.